ci/requirements-3.5-dev.yml

```diff
@@ -10,6 +10,9 @@ dependencies:
   - clickhouse-sqlalchemy
   - cmake
   - flake8
+  - gdal=2.2
+  - geoalchemy2
+  - geopandas
```

Comment:
It seems like maybe we shouldn't support python3.5 ... I'm actually pretty keen to remove support for it, so the fewer new backends/features that don't support it the better. Will it be a lot of additional work to avoid testing and the other things needed to do to make sure that this functionality doesn't work on 3.5?

Reply:
I'm not sure I understand your question. Do you want to deliberately make sure this doesn't work on 3.5?

Reply:
Yes, there's some nonzero maintenance burden associated with it.